packages/kit/src/core/adapt/prerender/fixtures/large-page/input.html

```diff
@@ -1786,6 +1786,16 @@ <h3 id="modules-$app-stores">$app/stores</h3>
 								>. It can be written to, but this will <em>not</em> cause changes to persist on the
 								server — this is something you must implement yourself.
 							</li>
+							<li>
+								<code>updating</code> is a
+								<a
+									href="https://svelte.dev/tutorial/writable-stores"
+									target="_blank"
+									rel="noopener noreferrer"
+									>writable store</a
+								>
+								whose initial value is false. The app's version will be checked every minute for changes, setting the value to true if the version does not match the version at the time the store was created. It can be written to when custom logic is required to detect updates. There is a <code>check</code> function to force the version check immediately, returning the result.
+							</li>
```

Comment:
we don't need to keep this file up to date. in fact i think we should remove it from the repo soon (it was originally there for benchmarking purposes)

```suggestion
```